driver/terraform_test.go

```diff
@@ -768,10 +832,67 @@ func TestInitTask(t *testing.T) {
 			} else {
 				assert.Error(t, err)
 			}
+
+			assert.NotEqual(t, tf.task.module, "./")
+			assert.NotEqual(t, tf.task.module, "../")
 		})
 	}
 }
 
+func TestTerraform_DestroyTask(t *testing.T) {
+	var w mocksTmpl.Watcher
+	tf := Terraform{
+		watcher: &w,
+	}
+	ctx := context.Background()
+
+	w.On("Deregister", mock.Anything).Return().Once()
+	tf.DestroyTask(ctx)
```

Comment:
Could we add an AssertExpectations here to check that deregister was called once?

Reply:
Ah, I thought `once` was sufficient, and that it meant it could only be called `once`. I misunderstood, good to know.

Reply:
I've added this in other places as well

Reply:
Yeah, it can only be called once, but it can also be called zero times! We don't seem to do AssertExpectations very often, but I had seen it here or there.